openTSNE/nearest_neighbors.py

```diff
@@ -371,7 +372,7 @@ def build(self, data, k):
 
         # Will use query() only for k>15
         if k <= 15:
-            n_neighbors_build = k + 1
+            n_neighbors_build = k
```

Comment:
Before, when we used `query` everywhere, that also returned the point itself as a nearest neighbor. When building the index the "proper" way, this is not the case, so I changed this to just `k`. It's still works fine for query later on.

Reply:
Hmm. Are you sure it's not the case? You still do `return indices[:, 1:], distances[:, 1:]` later on, so will `index.neighbor_graph` contain 16 elements when you build with `n_neighbors=15`?

Reply:
Yes, you were right. At first I thought I should just fix it to return `indices, distances` as is, but then I checked and the nearst neighbor graph actually returns the points themselves as their closest neighbors with distance 0. That's quite unexpected, but I've fixed it now.

Reply:
ok